src/test/regress/expected/text_search.out

```diff
@@ -0,0 +1,390 @@
+CREATE SCHEMA text_search;
+CREATE SCHEMA text_search2;
+SET search_path TO text_search;
+-- create a new configruation from scratch
+CREATE TEXT SEARCH CONFIGURATION my_text_search_config ( parser = default );
+CREATE TABLE t1(id int, name text);
+CREATE INDEX t1_search_name ON t1 USING gin (to_tsvector('text_search.my_text_search_config'::regconfig, (COALESCE(name, ''::character varying))::text));
```

Comment:
We might consider adding a simple test to arbitrary configs, it sometimes spots very interesting edge cases

Reply:
There is a thread lingering for me to add tests to this suite for many of the object propagation things. Am planning to address those after codefreeze.